### DISCUSSION TITLE
pkg_npm and Peer Dependencies

### DISCUSSION THREAD

**Discussion by flaviotsf** [bazelbuild/rules_nodejs#2533] [Q&A]
pkg_npm receives an array of dependencies. How would one signal that a dependency should be added to peer dependency?

A common example is a component library that needs Recat as a peer dependency vs. dependency.

**Comment by mattem** [comment]
The array of dependencies passed to `pkg_npm` on the `deps` attr are build dependencies, eg files that should be included in the package, this isn't related to the npm `package.json` dependencies, see https://bazelbuild.github.io/rules_nodejs/Built-ins.html#pkg_npm-deps
